lib/phoenix_live_view/upload_config.ex

```diff
@@ -95,6 +96,7 @@ defmodule Phoenix.LiveView.UploadConfig do
 
   @type t :: %__MODULE__{
           name: atom() | String.t(),
+          cid: :unregistered | nil | integer(),
```

Comment:
What is the difference between :unregistered and nil? Can you please drop a comment in the code?